src/CMakeLists.txt

```diff
@@ -56,8 +56,9 @@ add_library(xania_lib STATIC
         wiznet.cpp
         xania.cpp
         Descriptor.cpp Descriptor.hpp
-        string_utils.cpp string_utils.hpp)
-target_link_libraries(xania_lib crypt chat)
+        string_utils.cpp string_utils.hpp
+        info.hpp WrappedFd.cpp WrappedFd.hpp)
+target_link_libraries(xania_lib crypt chat CONAN_PKG::fmt)
```

Comment:
ok so an actual fmt shared lib we're linking with

Reply:
Right! 99.9% of fmt is headers but there's a nugget that's a library. Either way, to get access to the headers, we "link" against it. CMake then arranges for the right `-Ipath/to/fmt/headers`